src/com/lftechnology/java/training/alina/jdbc/views/AdminView.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.alina.jdbc.views;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+
+public class AdminView {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)